templates/zerver/emails/base.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+<head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Zulip</title>
+</head>
+<body>
+<table width="80%" style="align:center; max-width:800px" align="center">
+    <tr>
+        <td>
+        {#
+            <a href="{{ server_uri }}/hello">
+              <img style="max-height:75px; height:75px;"  height="75px" alt="{{ _('Zulip') }}" title="Zulip" src="{{ server_uri }}/static/images/landing-page/zulip-header.png" />
+            </a>
+        #}
+        {% block header %}{% endblock %}
```

Comment:
A follow-up commit or PR can be to remove this block, and have the CSS for all emails live in this base template.